### ISSUE TITLE
build.gradle not found

### ISSUE THREAD

**Issue by gazzaf** [Pushwoosh/pushwoosh-geozones-cordova-plugin#1]
Hi. When adding this plugin version 1.18.0 to 1.18.3 (I tried them all) to a Cordova 10 project with the Android 9.1.0 platform, it displays an error and the app fails to work correctly.

```
D:\myApp>cordova plugins add pushwoosh-geozones-cordova-plugin@1.18.3
Installing "pushwoosh-geozones-cordova-plugin" for android
Error during processing of action! Attempting to revert...
Failed to install 'pushwoosh-geozones-cordova-plugin': CordovaError: Uh oh!
"D:\myApp\plugins\pushwoosh-geozones-cordova-plugin\build.gradle" not found!
    at copyFile (D:\myApp\platforms\android\cordova\lib\pluginHandlers.js:208:36)
    at copyNewFile (D:\myApp\platforms\android\cordova\lib\pluginHandlers.js:233:5)
    at install (D:\myApp\platforms\android\cordova\lib\pluginHandlers.js:80:17)
    at ActionStack.process (D:\myApp\node_modules\cordova-common\src\ActionStack.js:56:25)
    at PluginManager.doOperation (D:\myApp\node_modules\cordova-common\src\PluginManager.js:112:24)
    at PluginManager.addPlugin (D:\myApp\node_modules\cordova-common\src\PluginManager.js:142:21)
    at D:\myApp\platforms\android\cordova\Api.js:161:78
Uh oh!
"D:\myApp\plugins\pushwoosh-geozones-cordova-plugin\build.gradle" not found!
```
This problem does not exist when using version 1.17.0. I've reported this via support but I thought to post it here in case anyone has experienced the same or is aware of a workaround? Thanks.
Gary.

**Comment by wfhm** [comment]
Hi Gary,

Got it reproduced, should've been fixed with 1.18.4, however, another issue appeared after the release. It looks like something happens when publishing to npm - local plugin version works just fine, but the npm version breaks the build. I'll update this thread when the npm release works as expected. Stay tuned!

**Comment by wfhm** [comment]
@gazzaf Fixed with the 1.18.5 release. Please let us know should there be any issues with the latest plugin version.

**Comment by gazzaf** [comment]
@wfhm Many thanks for fixing this so quickly, and also for fixing the main PW plugin by removing the Android ID stuff. It's all very much appreciated. 🥇 :-)
